packages/lab/src/slider/RangeSlider.tsx

```diff
@@ -23,6 +27,10 @@ export interface RangeSliderProps
    * Disable the slider.
    */
   disabled?: boolean;
+  /**
+   * Show visual ticks above the mark labels
```

Comment:
slight danger of over complicating the language by overusing the word label ?

In this description and some other references, we have marks and mark labels.
But we also have labels that go at the end of the tracks.

Should we have track, markers, labels and ticks?
and not refer to the mark labels or marks as labels ?